sandbox/mini-non-shiny-environment/07_system_overview.R

```diff
@@ -0,0 +1,75 @@
+
+system_df_prep <- EnrollmentAdjust %>%
+  left_join(Project0 %>%
+              select(ProjectID, ProjectName, OrganizationID, RRHSubType),
+            by = "ProjectID") %>%
+  left_join(Organization %>% select(OrganizationID, OrganizationName),
+            by = "OrganizationID") %>%
```

Comment:
I think you're only joining to Organization to get OrganizationName, but Project0 already has it (line 92 of initial data prep). So let's just add OrganizationName to the Project0 join